hack/test

```diff
@@ -107,7 +108,18 @@ if [ "$TEST_KEEP_CACHE" != "1" ]; then
   trap 'docker rm -v $cacheVolume' EXIT
 fi
 
-baseCreateFlags="--rm --privileged \
+dockerConfigMount=""
+if [ "$GITHUB_ACTIONS" = "true" ] || [ -n "$MOUNT_BUILDKIT_DOCKER_CONFIG_PATH" ]; then
```

Comment:
```suggestion
if [ "$GITHUB_ACTIONS" = "true" ]; then
```

Reply:
? I intentionally made it so that it can be used locally as well, but only if custom path is set. The default path is not used locally.

Reply:
But we don't set `MOUNT_BUILDKIT_DOCKER_CONFIG_PATH` in GitHub Actions, I would assume `MOUNT_BUILDKIT_DOCKER_CONFIG_PATH` was meaningful at https://github.com/moby/buildkit/pull/5390/files#diff-ecedb817abdf33fd9ee7bb0c35c9cfcedfb39d8f3b261d92a6254066578aff08R114

Reply:
We don't need to set it because the default path from $HOME is used there.

Reply:
Nevermind it's a nor :see_no_evil: